gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/components/LazyAvatarList.kt

```diff
@@ -36,9 +38,14 @@ internal fun LazyAvatarRow(
 ) {
     var parentBounds by remember { mutableStateOf(Rect(Offset.Zero, Size.Zero)) }
 
+    val density = LocalDensity.current
+    val layoutDirection = LocalLayoutDirection.current
+
     LazyRow(
         horizontalArrangement = horizontalArrangement,
-        modifier = modifier.onGloballyPositioned { coordinates -> parentBounds = coordinates.boundsInRoot() },
+        modifier = modifier.onGloballyPositioned { coordinates ->
+            parentBounds = coordinates.boundsInRoot().getPaddedBounds(contentPadding, density, layoutDirection)
+        },
```

Comment:
⛏️ Maybe I'm nitpicking, but I think for the horizontal list we could have no padding because the Avatar themselves can be drawn from edge to edge of the Parent composable. 

We use `contentPadding` on the `LazyRow` which means only the first and last element will have the padding applied, but when you scroll it will look like there's no padding. 

<img width="534" alt="Screenshot 2024-11-22 at 10 31 09" src="https://github.com/user-attachments/assets/f0b3a0de-b4d8-4f81-80d5-31d5990ddb66">

Reply:
But then the first element will still have padding because it's aligned with the `Start` of the avatar, which is drawn with padding. And for the last element, the popup won't have padding at the `End`, although the last avatar has padding.

Reply:
Hmm not sure I'm following. The Popup is drawn also based on the anchor (Avatar) so with the `Alignment.Start` it should be drawn properly for the first element even if it has some extra padding.

Reply:
Exactly, so in regard to the list itself it has padding.

Reply:
Good point @AdamGrzybkowski . I agree that we can remove the padding for the horizontal list.

Reply:
Alright, if you both agree, I'll do it! 😃

Reply:
Then the util function won't be needed outside of the vertical list, so we can make it private to that file